tests/utils.py

```diff
@@ -74,6 +74,14 @@ def gen_attr_names():
             yield outer + inner
 
 
+def maybe_underscore_prefix(source):
+    """A generator to sometimes prepend an underscore."""
```

Comment:
*cough* `"""` on separate lines *cough*

I should really look into pylint so I don’t get to be the asshole that pesters everyone. :)